src/Template/Element/Form/children.twig

```diff
@@ -39,11 +39,16 @@
                             <span class="has-text-size-smaller prop-id"><: related.id :></span>
                         </header>
 
-                        <div class="thumbnail" v-if="related.meta.thumb_url">
-                            <figure>
+                        <div class="thumbnail" v-if="related.type === 'images'" >
+                            <figure v-if="related.meta && related.meta.thumb_url">
```

Comment:
Make sure to use always `related.meta && related.meta.thumb_url` as condition, not just `related.meta.thumb_url`